pkg/collector/corechecks/ebpf/oomkill/oom_kill.go

```diff
@@ -115,9 +115,9 @@ func (m *OOMKillCheck) Run() error {
 			log.Debugf("Unable to extract containerID from cgroup name: %s, err: %v", line.CgroupName, err)
 		}
 
-		entityID := types.NewEntityID(types.ContainerID, containerID).String()
+		entityID := types.NewEntityID(types.ContainerID, containerID)
 		var tags []string
-		if entityID != "" {
+		if entityID.Empty() {
```

Comment:
This change looks counter-intuitive to me because I would have expected `""` to be equivalent to `.Empty()` and not the other way round.

Shouldn’t this be:
```suggestion
		if entityID.NotEmpty() {
```
or
```suggestion
		if !entityID.Empty() {
```
?